pages/linux/minicom.md

```diff
@@ -0,0 +1,16 @@
+# minicom
+
+> A program to do a serial communication with a device.
```

Comment:
probably lets throw in the `interface` as well to make it absolutely obvious: (thoughts?)

```suggestion
> A program to communicate with the serial interface of a device.
```

Reply:
I think the `interface` could be a virtual as well :)

Reply:
I see... Okay, this will do. Thanks for the suggestion :)